MIDAS/src/sensor_data.h

```diff
@@ -63,8 +63,19 @@ struct Barometer {
     float pressure = 0;
 };
 
+// Detects if a charge or 
 struct Continuity {
-    bool is_continuous = false;
+    Continuity() = default;
+    bool pins[6];
+    // fancy operator overloading, you CS 128 kids won't get it until like week 6, and even then I don't think y'all
+    // mess with overloading []
+    bool& operator[](const int i) {
```

Comment:
I don't think we need this overloading, we can just access the pins field itself. It seems a little overcomplicated